### PR TITLE
Added validation of requests

### PR DESCRIPTION
**Pull request by BrilStrawhat**

### CODE REVIEW

File: server/inc/server.h [hunk 1]
```diff
@@ -14,7 +14,7 @@ typedef struct s_client {
     GOutputStream *ostream;
     GDataInputStream *data_in;
     GDataOutputStream *data_out;
-    gint64 uid;
+    gint32 uid;
```
Comment (Hvvang) [inline server/inc/server.h:17]: It's better to remove this and add auth_token that also will need for another feature

Comment (BrilStrawhat) [reply server/inc/server.h:17]: No, it would be in new variable in structure

Comment (Hvvang) [reply server/inc/server.h:17]: why is uid so important for u? IMHO, it's trash.

**Review by Hvvang — APPROVED** [review]
Everything ok

**Pull request merged**